test/SourceKit/Misc/cancellation.swift

```diff
@@ -0,0 +1,13 @@
+// Check that we can cancel requests.
+// We need to wait a little bit after request scheduling and cancellation to make sure we are not cancelling the request before it got scheduled.
+
+// RUN: not %sourcekitd-test -req=cursor -id=slow -async -pos=10:3 %s -- %s == \
+// RUN: -shell -- sleep 1 == \
```

Comment:
I thought this would need `// REQUIRES: shell`, but I guess the tests passed on all platforms :shrug: